src/recommendations/ProductCard/BaseCard/index.jsx

```diff
@@ -16,12 +16,14 @@ const BaseCard = ({
   footer,
   handleOnClick,
   isLoading,
+  redirectUrl,
 }) => (
   <div className="recommendation-card" key={`container-${uuid}`}>
     <Hyperlink
       target="_blank"
       className="card-box d-inline"
       showLaunchIcon={false}
+      destination={redirectUrl}
```

Comment:
I think destination will cause redirection to redirectUrl. Does this mean that there will be cases where redirection will be performed before handleOnClick is executed?

Reply:
I have added `event.preventDefault()` in the `handlOnClick` event. It will prevent it from redirecting. The redirection logic will take place inside `trackRecommendationClick()` as it currently does.